src/App.js

```diff
@@ -125,6 +128,44 @@ const App = () => {
     setIsAlertOpen(true);
   };
 
+  const editExpense = async (id, newExpenseData) => {
```

Comment:
this method seems out of place. usually this would belong into some state/persistence container (e.g. redux). let's have a discussion about this in the chat sometime. my goal would be that all state-related code is in one place.

Reply:
perfect. I would love to see this state logic to move to a separate class/file.